src/components/AsideHeader/components/AllPagesPanel/AllPagesPanel.module.scss

```
@@ -5,11 +5,30 @@
     height: 100%;
     padding: var(--g-spacing-4) var(--g-spacing-6);
     box-sizing: border-box;
-    --gn-aside-header-item-expanded-radius: #{variables.$item-expanded-radius};
-    --_--item-height: #{variables.$item-height};
 
     &__content {
         overflow: auto;
         flex: 1;
+
+        margin: 0 calc(var(--g-spacing-6) * -1);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)